bots/README.md

```diff
@@ -31,7 +31,7 @@ For more details, see the [Bots Architecture section](bots-architecture/#bot-pla
 
 These are the basic steps for using bots with Rocket.Chat:
```

Comment:
Please change ...  "Bots (or Chatbots) are pieces of software that provide automated messaging in a chat platform. They are useful for running pre-defined (scripted) interactions to provide support, conversational workflow or on-boarding with multi-channel, multi-user, public and private interactions. Bots can also be integrated with external services."

to ...

"Bots (or Chatbots) are pieces of software or software systems that provide automated messaging to a chat platform. They typically add value by integrating large external services such as CRM and big data sources.  They provide a friendly conversational interface that users on chat platforms are already familiar with to these complex systems; add adds automation, natural language understanding and machine learning possibilities. "